rocketmq/exceptions.py

```diff
@@ -38,9 +38,9 @@ def ffi_check(status_code):
     msg = dll.GetLatestErrorMessage()
     if msg is not None:
         msg = msg.decode('utf-8')
-        msg = re.sub('<.*?(rocketmq-client-cpp/)(.*)>', '\\1\\2', msg)
-        if msg.startswith('msg: '):
-            msg = msg[5:]
+        # msg = re.sub('<.*?(rocketmq-client-cpp/)(.*)>', '\\1\\2', msg)
```

Comment:
This makes error message more friendly, I'd say the error message produced by rocketmq-client-cpp is kind of weird

1. No need to prefix error message with `msg: `
2. Avoid absolute file path because it usually does not match user's file path.